src/renderer/src/routes/pages/Welcome/SecuritySetup/Step2.tsx

```diff
@@ -57,11 +58,13 @@ export const WelcomeSecuritySetupStep2Page = ({ onSubmit }: TProps) => {
       mnemonic: words.join(' '),
     })
 
-    await createStandardAccount({
-      wallet,
-      blockchain: 'neo3',
-      name: commonT('account.defaultName', { accountNumber: 1 }),
-    })
+    for (const blockchain of blockchainNames) {
+      await createStandardAccount({
+        wallet,
+        blockchain,
+        name: commonT('account.defaultName', { accountNumber: 1 }),
+      })
+    }
```

Comment:
It would be more performant to use Promise.all, as the number of blockchains tends to grow
```suggestion
   const promises = blockchainNames.map(blockchain => createStandardAccount({
        wallet,
        blockchain,
        name: commonT('account.defaultName', { accountNumber: 1 }),
      }))
      
      await Promise.allSettle(promises)
```